edx_proctoring/api.py

```diff
@@ -1404,7 +1404,7 @@ def update_attempt_status(attempt_id, to_status,
         # only proctored/practice exams have a backend
         # timed exams have no backend
         backend_method = None
-        if to_status == ProctoredExamStudentAttemptStatus.started:
+        if to_status == ProctoredExamStudentAttemptStatus.started and add_start_time:
```

Comment:
add_start_time is defined earlier in the function as
add_start_time = (to_status == ProctoredExamStudentAttemptStatus.started and not exam_attempt_obj.started_at )

It should have a value of true only if the attempt has never transitioned to started before.

Reply:
#NIT please add a comment to the code here like how you added this explanation to the PR.